rust/private/rustc.bzl

```diff
@@ -1,8 +1,8 @@
 """
-Rules for interfacing with the rust toolchain: rustc and rustdoc.
+Rules for interfacing with rustc.
 """
 
-load(":utils.bzl", "relative_path")
+load(":private/utils.bzl", "find_toolchain", "relative_path")
```

Comment:
This isn't right, right?

The file here is private/rustc.bzl, and you're trying to load a file "private/private/utils.bzl"?

Perhaps I'm missing something. Similar for the below files

Reply:
This works, but I'm not sure why; I guess it's relative to the package dir?

I tried changing it to absolute paths and it's unhappy with that, so I am going to punt on this.